actors/migration/nv16/system.go

```diff
@@ -0,0 +1,29 @@
+package nv16
+
+import (
+	"context"
+
+	system8 "github.com/filecoin-project/specs-actors/v8/actors/builtin/system"
+	"github.com/ipfs/go-cid"
+	cbor "github.com/ipfs/go-ipld-cbor"
+)
+
+// System Actor migrator
+type systemActorMigrator struct {
+	OutCodeCID   cid.Cid
+	ManifestData cid.Cid
+}
+
+func (m systemActorMigrator) migrateState(ctx context.Context, store cbor.IpldStore, in actorMigrationInput) (*actorMigrationResult, error) {
+	state := system8.State{BuiltinActors: m.ManifestData}
```

Comment:
nit: If you give the migrator the manifest data itself you can write the data to the store here for extra clarity that we're not writing any unreachable ids

Reply:
Maybe.

But the data should already be in the store, thats where we got them in the first place.

I will add a comment.